Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -859,6 +859,7 @@ void PcapLiveDevice::setDeviceMtu()
 	}
 
 	m_DeviceMtu = ifr.ifr_mtu;
+	::close(socketfd);
```

Comment:
Maybe we should refactor this code to make sure the socket always closes?
I mean something like this:
```cpp
int socketfd = -1;
try
{
	int socketfd = socket(AF_INET, SOCK_DGRAM, IPPROTO_IP);
	...
}
catch (const std::exception& e)
{
 	//log exception using e.what()
}
if (socketfd != -1)
{
	::close(socketfd);
}
```

Reply:
fixed.